.github/.kodiak.toml

```diff
@@ -13,6 +13,7 @@ require_automerge_label = true
 
 [merge.message]
 title = "pull_request_title"
-body = "empty"
+body = "pull_request"
```

Comment:
Ich finde dies auch die bessere einstellung, da einige unsere commits eher nackt wirken und context aus den PRs der eigentlichh vorhanden ist, nicht mitgenommen wird. 

Beispiel https://github.com/redaxo/redaxo/commit/aa2cc24fc5561cd8a669d7f94dc36e86201e642a

Wenn man bedenkt dass man ggf irgendwann mal den hoster wechseln könnte und somit nicht mehr bei github wäre, wären die infos alle im GIT und man wäre nicht abhängig davon dass man auch Pull Request Daten mit umziehen müsste

Reply:
> Ich finde dies auch die bessere einstellung, da einige unsere commits eher nackt wirken und context aus den PRs der eigentlichh vorhanden ist, nicht mitgenommen wird.

Das stimmt schon. Aber ich finde, oftmals ist der PR-Eröffnungsbeitrag nicht wirklich für einen Commit-Body geeignet. Entweder weil er verhaltet ist, und sich im Laufe des PRs wesentliche Dinge geändert haben, oder auch einfach von der Art der Formulierung. Es ist nun mal eigentlich in der Einstieg in die Diskussion zu einem Vorschlag, und nicht das Ende.

Ich habe mal ein paar Beispiele rausgesucht:
https://github.com/redaxo/redaxo/pull/3587: "Die Namen finde ich immer noch schwierig. Mein Vorschlag ist nun..." finde ich nicht gut für den Commit am Ende
https://github.com/redaxo/redaxo/pull/3486: Die Keys hießen am Ende ganz anders.
https://github.com/redaxo/redaxo/pull/3671: "hab ich mal nen PR aufgemacht" und "feel free to close."
https://github.com/redaxo/redaxo/pull/3657: "Wenn der PR so (oder mit Änderungen) angenommen wird, würde ich den Extension Point danach noch in der Dokumentation ergänzen."

Mich würde es stören, wenn diese Beschreibungen in der Haupthistory in den Commits wären.

> Wenn man bedenkt dass man ggf irgendwann mal den hoster wechseln könnte und somit nicht mehr bei github wäre, wären die infos alle im GIT und man wäre nicht abhängig davon dass man auch Pull Request Daten mit umziehen müsste

Ja, das wäre in der Tat ein Vorteil dabei!
(Wobei ich einen veralteten Eröffnungsbeitrag, wo man dann gar nicht mehr zu der Gesamtdiskussion kommen könnte wegen Hosterwechsel, unter Umständen verwirrender fände, als gar keinen Commit-Body.)